docs/common/input.md

```diff
@@ -415,7 +415,9 @@ time series data (and excluding checkpoint data). Relevant parameters are
     * `binary`: output is written using PETSc's binary data format
     * `xdmf`: output is written to the [XDMF](https://xdmf.org/index.php/XDMF_Model_and_Format) format
     * `cgns`: output is written to the [CFD General Notation System (CGNS)](https://cgns.github.io/) format
-* `interval`: the number of time steps between output dumps. Default value: 0 (no output)
+* `step_interval`: the number of time steps between output dumps. Default value: 0 (no output)
+* `time_interval`: the temporal frequency between output dumps.Default value: 0 (no output)
+* `time_unit`: units of temporal frequency output.
```

Comment:
is `time_unit` required when `time_interval` is given? Or does it have a default?

Reply:
Also: should we be allowing/encouraging the use of multiple time units in single input files? I know PFLOTRAN allows this because its input format associates units with each quantity, but without such a feature this could get confusing.

Reply:
Good catch, as I forgot to add a check if `time_unit` is specified or not.